client/my-sites/current-site/index.jsx

```diff
@@ -85,21 +85,21 @@ module.exports = React.createClass( {
 		return this.props.sites.getSelectedSite();
 	},
 
-	getDomainExpirationNotices: function() {
+	getDomainWarnings: function() {
 		let domainStore = this.state.domainsStore.getBySite( this.getSelectedSite().ID ),
 			domains = domainStore && domainStore.list || [];
 		return (
 			<DomainWarnings
 				selectedSite={ this.getSelectedSite() }
 				domains={ domains }
-				ruleWhiteList={ [ 'expiredDomains', 'expiringDomains' ] } />
+				ruleWhiteList={ [ 'expiredDomains', 'expiringDomains', 'wrongNSMappedDomains' ] } />
 		);
 	},
 
 	getSiteNotices: function() {
 		return (
 			<div>
-				{ this.getDomainExpirationNotices() }
+				{ this.getDomainWarnings() }
```

Comment:
Can you move this to the `site/notice.jsx` sub component and render it there? The design was tweaked to take less space and should be the canonical place to render site notices.